src/Agent.Worker/Build/TeeTFCommandManager.cs

```diff
@@ -47,14 +48,14 @@ public sealed class TeeTFCommandManager : AgentService, ITeeTFCommandManager
         {
             ArgUtil.NotNullOrEmpty(version, nameof(version));
             ArgUtil.NotNullOrEmpty(directory, nameof(directory));
-            await RunCommandAsync("get", $"-version:{version}", "-recursive", "-overwrite", directory);
+            await RunCommandAsync(FormatFlags.None, "get", $"-version:{version}", "-recursive", "-overwrite", directory);
         }
 
         public string ResolvePath(string workspace, string serverPath)
         {
             ArgUtil.NotNullOrEmpty(workspace, nameof(workspace));
             ArgUtil.NotNullOrEmpty(serverPath, nameof(serverPath));
-            string localPath = RunPorcelainCommandAsync("resolvePath", $"-workspace:{workspace}", serverPath).GetAwaiter().GetResult();
+            string localPath = RunPorcelainCommandAsync(FormatFlags.None, "resolvePath", $"-workspace:{workspace}", serverPath).GetAwaiter().GetResult();
```

Comment:
async/await?
I understand this will cause changing the interface of sourceprovider. not sure what might be good to do.